roles/tests-integration/tests/common/mod.rs

```diff
@@ -1,13 +1,27 @@
+mod sniffer;
+
 use bitcoind::{bitcoincore_rpc::RpcApi, BitcoinD, Conf};
 use flate2::read::GzDecoder;
+use key_utils::{Secp256k1PublicKey, Secp256k1SecretKey};
+use once_cell::sync::Lazy;
+use pool_sv2::PoolSv2;
+use sniffer::Sniffer;
 use std::{
+    collections::HashSet,
+    convert::TryFrom,
     env,
     fs::{create_dir_all, File},
     io::{BufReader, Read},
+    net::{SocketAddr, TcpListener},
     path::{Path, PathBuf},
+    str::FromStr,
+    sync::Mutex,
 };
 use tar::Archive;
 
+// prevents get_available_port from ever returning the same port twice
+static UNIQUE_PORTS: Lazy<Mutex<HashSet<u16>>> = Lazy::new(|| Mutex::new(HashSet::new()));
+
 const VERSION_TP: &str = "0.1.7";
```

Comment:
I would use the latest version release, which seems the most stable, `0.1.9`: https://github.com/Sjors/bitcoin/releases/tag/sv2-tp-0.1.9